src/ui/newmappopup.cpp

```diff
@@ -164,6 +171,86 @@ void NewMapPopup::setDefaultValues(int groupNum, QString mapSec) {
     }
 }
 
+void NewMapPopup::setDefaultValuesImportMap(MapLayout *mapLayout) {
+    ui->lineEdit_NewMap_Name->setText(project->getNewMapName());
```

Comment:
Most of this function is copy-pasted defaults from the other function in this class.  Let's move those shared defaults into a single function.